client/app/queue/components/CompleteTaskModal.jsx

```diff
@@ -442,6 +454,23 @@ class CompleteTaskModal extends React.Component {
     return formattedInstructions;
   };
 
+  validateCGForm = () => {
```

Comment:
```suggestion
  validateCaregiverForm = () => {
```
suggestion for improved readability